### PR TITLE
Upgrade closure to fix a reported security issue in one of its dependencies

### PR DESCRIPTION
**Pull request by kripken**
Also seems to help code size a tiny bit.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   },
   "dependencies": {
     "google-closure-compiler": "20191111.0.0",
-    "html-minifier-terser": "5.0.2"
+    "html-minifier-terser": "5.0.2",
+    "minimist": "1.2.2"
```
Comment (sbc100) [inline package.json]: I don't understand.  Was this coming in indirectly before?  Via what?  I rather update terser if that is what is pulling it in.

Comment (sbc100) [reply package.json]: At least it would be nice to have a commit here saying that we don't directly use this.

Comment (kripken) [reply package.json]: Looks like this was used by closure. It's possible upgrading closure would upgrade it's deps? That would be a bigger change though.

I don't think it's valid to add comments to JSON files. My text editor even shows an error when I try.

Comment (sbc100) [reply package.json]: I think updating google-closure-compiler is probably the only sensible fix then.  Fixing it this way is a little confusing and I'm not sure these types of security issues are a problem for emscripten.

**Comment by kripken** [comment]
I pushed an upgrade of closure instead, let's see if that works...

**Comment by kripken** [comment]
Tests look good, so upgrading closure may be the right thing here.

**Pull request merged**